src/openmrs-esm-error-handling.ts

```diff
@@ -9,6 +9,8 @@ export function handleApiError() {
   };
 }
 
-window.addEventListener("error", info => {
-  // call "showToast()" or similar inside of openmrs-esm-styleguide
-});
+window.onerror = function(info) {
+  showToast(info);
```

Comment:
Now that showToast takes in an object, we should update this, too. Also, the message should be a generic message that leets the user know that an error has occurred.

```suggestion
  showToast({description: "Oops! An unexpected error occurred."})
```

Reply:
Looks good to me with addition of @joeldenning's suggestion